cleverpush/src/main/java/com/cleverpush/banner/AppBannerModule.java

```diff
@@ -816,6 +825,34 @@ void scheduleFilteredBanners() {
     }
   }
 
+  private boolean checkIsEveryTrigger(Banner banner, boolean isEveryTrigger) {
+    String bannerEventId = null;
+    boolean isConditionTrue = false;
+    if (isEveryTrigger && banner.getTriggerType() == BannerTriggerType.Conditions) {
+      for (BannerTrigger trigger : banner.getTriggers()) {
+        for (BannerTriggerCondition condition : trigger.getConditions()) {
+          if (condition.getType() != null) {
```

Comment:
if (condition.getType() == null
|| !condition.getType().equals(BannerTriggerConditionType.Event)
|| condition.getEvent() == null) {
  continue;
}
and remove these two if condition